torch/_functorch/aot_autograd.py

```diff
@@ -3742,6 +3744,10 @@ def aot_export_module(
     # If trace_joint is True, we expect your module to return a scalar loss.
     # Your module can return multiple outputs, so you must specify which output the loss is.
     output_loss_index: Optional[int] = None,
+    # For debug purposes the user may want to inspect the joint graph.
+    # Skipping the flatten_joint stage perserves the node.meta information
+    # available in the joint fwd/bwd graph fx_g.
+    skip_flatten_joint: int = False,
```

Comment:
Like a few others have mentioned - I'm not a huge fan of (yet) another config in AOTAutograd. Would it be possible to:

It's a bit unclear to me what purpose this serves:
(1) this flag will force you to only generate a forward graph? Why not just pass `trace_joint=False` in your example?
(2) more of general question: is important/useful to use the profiler with `torch.export`, which is what your testing in the the test case from `test_aotdispatch.py`? Or is the benefit mostly from working with `torch.compile` which doesn't seem like it needs any changes).

Reply:
Thanks for the feedback @bdhirsh .  I went ahead and removed the changes I made to aot_export_module().  I originally introduced these changes because I needed a way to observe from code that the sequence_nr() was being assigned correctly to each op.  aot_export_module() allowed me to do this, but only if I forced the control flow to bypass a call to make_fx().  I found a way to make the test case cover this.  The intent of the sequence_nr() changes is that the profiler or developer can observe the sequence_nr() from downstream inductor (torch/_inductor/scheduler.py   origin_ops).